pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+// EnforceHostNetwork is a global/package variable, that can be set via the rook-operator-config setting
+// "ROOK_ENFORCE_HOST_NETWORK". when set to "true", it lets rook create all pods with host network enabled. This can be used, for example, to help allow Rook to run in k8s clusters with no CNI, so that host networking is required
+
```

Comment:
nit: no blank line between the comment and the var

Reply:
gone